idaes/surrogate/pysmo/tests/test_sampling.py

```diff
@@ -189,6 +194,8 @@ def test_points_selection_01(self, array_type):
         input_array = array_type(self.test_data_3d)
         generated_sample_points = np.array([[-0.5, 10], [10, 100]])
         SamplingClass = SamplingMethods()
```

Comment:
Here it's not clear if `SamplingClass` is (A) a class object, or (B) an instance of the class `SamplingMethod`. If (B), I'd suggest to rename it to e.g. `sampling_methods` as `SamplingClass` strongly implies (A) (both in the name itself and the so-called `PascalCase` formatting)

Reply:
Addressed.